src/packages/engine-core/src/NAPIEngine.ts

```diff
@@ -190,8 +177,14 @@ You may have to run ${chalk.greenBright(
         try {
           this.QueryEngine = require(this.libQueryEnginePath).QueryEngine
         } catch (e) {
-          // How should we handle this
-          throw new Error(e)
+          throw new PrismaClientInitializationError(
+            `Unable to load NAPI Library ${
+              this?.libQueryEnginePath
```

Comment:
```suggestion
              this.libQueryEnginePath
```
Small edit: The conditional seems not necessary to me
Thanks for improving it!

Reply:
You're right. Will remove it